### ISSUE TITLE
Inconsistent sensuctl silenced CLI behavior 

### ISSUE THREAD

**Issue by agoddard** [sensu/sensu-go#3811]
## Expected Behavior
`sensuctl silenced create` should prompt the user interactively for input when no arguments are supplied.

## Current Behavior
`sensuctl silenced create` with no arguments throws an error if required arguments are not supplied, however `sensuctl silcenced delete` presents the user with interactive steps.

## Possible Solution
Update `sensuctl silenced create` to prompt for interactive input when no arguments are supplied.

## Steps to Reproduce (for bugs)
```
 ~  sensuctl silenced create
Error: required flag(s) "reason" not set
 ~  sensuctl silenced delete
? Subscription:
```

**Comment by portertech** [comment]
With all other command, you have to specify `--interactive` to get the field prompts. Caleb had the idea to add a hint to specify it.

**Comment by calebhailey** [comment]
Yes, in general most of the `sensuctl` commands will error if no arguments are provided, and those error messages generally don't give you the clue that a `--interactive` flag exists. I think we should at least hint that `--interactive` exists, if not just go ahead and enable `--interactive` mode (i.e. if no required arguments are provided, prompt the user via `--interactive` mode).
